javatests/arcs/core/data/proto/HandleProtoDecoderTest.kt

```diff
@@ -102,11 +106,16 @@ class HandleProtoDecoderTest {
             listOf("PERSISTENT", "QUERYABLE")
         )
         val handleProto = parseHandleProtoText(handleText)
-        with(handleProto.decode()) {
+
+        val handles = mapOf(
+            "handle1" to Handle("handle1", Handle.Fate.MAP, TypeVariable("handle1")),
+            "handle_join" to Handle("handle_join", Handle.Fate.JOIN, TypeVariable("handle_join"))
+        )
+        with(handleProto.decode(handles)) {
             assertThat(name).isEqualTo("thing")
             assertThat(fate).isEqualTo(Handle.Fate.JOIN)
             assertThat(storageKey).isEqualTo("ramdisk://b")
-            assertThat(associatedHandles).isEqualTo(listOf("handle1", "handle_join"))
+            assertThat(associatedHandles.map { it.name }).isEqualTo(listOf("handle1", "handle_join"))
```

Comment:
```suggestion
            assertThat(associatedHandles.map { it.name }).isEqualTo(listOf(handles["handle1"], handles["handle_join"]))
```

Reply:
I see what you mean; I'll also remove the `map`.